airbrakes/data_handling/processed_data_packet.py

```diff
@@ -9,6 +9,8 @@ class ProcessedDataPacket(msgspec.Struct):
     data.
     """
 
-    avg_acceleration: tuple[float, float, float]
-    current_altitude: float
-    speed: float
+    # pro for processed, I don't like the abbreviation, but it's better consistent est
```

Comment:
"consistent est" consistent what now? Also when I read "pro" initially I had no idea what that was supposed to mean lol